src-java/base-topology/base-storm-topology/src/main/java/org/openkilda/wfm/AbstractBolt.java

```diff
@@ -45,7 +45,7 @@
 
     protected transient Logger log = makeLog();
 
-    protected transient boolean active;
+    protected boolean active = false;
```

Comment:
Because we should not loose it during topology upload. Its current value "false" allow to lose it, but in general we should not loose it. Same reason to obviously declare false value, just for readability.